css/styles.css

```diff
@@ -50,15 +52,82 @@ li {
   background-color: #e5e5e5;
 }
 
+.close-menu {
```

Comment:
- [ ] The menu does not take all the space of the page, please consider using 100% ```width``` and ```height``` 

![Screenshot_20220512_061115](https://user-images.githubusercontent.com/25227022/167996465-b8a78ded-1a65-4f99-96fc-f05f2b88259e.png)